test/functional/wallet_listsinceblock.py

```diff
@@ -40,13 +40,15 @@ def run_test(self):
     def test_no_blockhash(self):
         txid = self.nodes[2].sendtoaddress(self.nodes[0].getnewaddress(), 1)
         blockhash, = self.nodes[2].generate(1)
+        blockheight = self.nodes[2].getblockheader(blockhash)
```

Comment:
To get the height, the field `"height"` from the RPC `getblockheader`s response has to be accessed, which is missing here.

Reply:
Fixed.